.github/workflows/bundle_with_dakota_macos.yml

```diff
@@ -22,50 +22,69 @@ on:
       INSTALL_DIR:
         required: true
         type: string
-      SEBA_TAG:
-        required: true
-        type: string
 
 jobs:
   build_dependencies:
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
was this intended for dev or production?

Reply:
This made most sense when developing, but I see why it would make more sense to do a full stop if something fails in production. Lets flip this to `true`.